### PR TITLE
Update feature-requests.md

### PR DESCRIPTION
**Pull request by wiersgallak**
Removed references of the Jira Customer Request Board

### CODE REVIEW

File: operations/research-and-development/product/product-planning/feature-requests.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ At Mattermost, we have two main places feature requests are tracked.
 
 The first one is [Uservoice](https://mattermost.uservoice.com/forums/306457-general). Uservoice is open to the public, and anyone is able to submit a request or upvote existing requests. We often send community members to this site when they request features. 
 
-The second place is in a "Customer Request" Jira project. This is an internal Jira project, so only Mattermost Staff can view it. Anytime a customer or potential customer has a request, it should be added to this project.
+The second place is in Productboard. Productboard is an internal tool used for tracking and prioritizing feature reqeusts. Customer-facing teams can add notes to Productboard based on [this process](https://handbook.mattermost.com/operations/research-and-development/product/how-to-guides-for-product/how-to-use-productboard#2-day-in-the-life-of-a-customer-support-manager-customer-success-manager-or-solution-architect).
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-planning/feature-requests.md]:
```suggestion
The second place is in ProductBoard. ProductBoard is an internal tool used for tracking and prioritizing feature reqeusts. Customer-facing teams can add notes to ProductBoard based on [this process](https://handbook.mattermost.com/operations/research-and-development/product/how-to-guides-for-product/how-to-use-productboard#2-day-in-the-life-of-a-customer-support-manager-customer-success-manager-or-solution-architect).
```

File: operations/research-and-development/product/product-planning/feature-requests.md [hunk 2]
```diff
@@ -10,9 +10,9 @@ At Mattermost, we have two main places feature requests are tracked.
...
 
-The product management team uses the information from both Uservoice and Jira to help prioritize features for the roadmap. 
+The Product Management team uses the information from both Uservoice and Productboard to help prioritize features for the roadmap.
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-planning/feature-requests.md]:
```suggestion
The Product Management team uses the information from both Uservoice and ProductBoard to help prioritize features for the roadmap. 
```

**Comment by justinegeffen** [comment]
I've changed the casing of Productboard to ProductBoard to make it consistent with the current documentation. Changes to the casing will be in a separate PR.

**Comment by wiersgallak** [comment]
> I've changed the casing of Productboard to ProductBoard to make it consistent with the current documentation. Changes to the casing will be in a separate PR.

I see on their website, it's all lowercase 'productboard'.  Not sure if we want to be consistent with how they do it.  To me, all lowercase looks incorrect, like our documentation is wrong.

**Comment by justinegeffen** [comment]
> > I've changed the casing of Productboard to ProductBoard to make it consistent with the current documentation. Changes to the casing will be in a separate PR.
> 
> I see on their website, it's all lowercase 'productboard'. Not sure if we want to be consistent with how they do it. To me, all lowercase looks incorrect, like our documentation is wrong.

Totally agree! I initially left it as-is based on our current docs and in a previous PR I noted the correct casing which I would do in a new PR. However, I'm more than happy to correct it here and then apply it across the rest of the handbook.

File: operations/research-and-development/product/product-planning/feature-requests.md [hunk 3]
```diff
@@ -10,9 +10,9 @@ At Mattermost, we have two main places feature requests are tracked.
 
 The first one is [Uservoice](https://mattermost.uservoice.com/forums/306457-general). Uservoice is open to the public, and anyone is able to submit a request or upvote existing requests. We often send community members to this site when they request features. 
 
-The second place is in a "Customer Request" Jira project. This is an internal Jira project, so only Mattermost Staff can view it. Anytime a customer or potential customer has a request, it should be added to this project.
+The second place is in ProductBoard. ProductBoard is an internal tool used for tracking and prioritizing feature reqeusts. Customer-facing teams can add notes to ProductBoard based on [this process](https://handbook.mattermost.com/operations/research-and-development/product/how-to-guides-for-product/how-to-use-productboard#2-day-in-the-life-of-a-customer-support-manager-customer-success-manager-or-solution-architect).
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-planning/feature-requests.md]:
```suggestion
The second place is in productboard. Productboard is an internal tool used for tracking and prioritizing feature reqeusts. Customer-facing teams can add notes to productboard based on [this process](https://handbook.mattermost.com/operations/research-and-development/product/how-to-guides-for-product/how-to-use-productboard#2-day-in-the-life-of-a-customer-support-manager-customer-success-manager-or-solution-architect).
```

File: operations/research-and-development/product/product-planning/feature-requests.md [hunk 4]
```diff
@@ -10,9 +10,9 @@ At Mattermost, we have two main places feature requests are tracked.
...
 
-The product management team uses the information from both Uservoice and Jira to help prioritize features for the roadmap. 
+The Product Management team uses the information from both Uservoice and ProductBoard to help prioritize features for the roadmap.
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-planning/feature-requests.md]:
```suggestion
The Product Management team uses the information from both Uservoice and productboard to help prioritize features for the roadmap. 
```

**Review by justinegeffen — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
@wiersgallak, I've submitted https://github.com/mattermost/mattermost-handbook/pull/345 to address your point re: naming conventions.

**Review by lfbrock — APPROVED** [review]

**Pull request merged**